src/app/modules/submission/email-submission/__tests__/email-submission.service.spec.ts

```diff
@@ -0,0 +1,317 @@
+import { pick } from 'lodash'
+
+import { BasicField, MyInfoAttribute } from 'src/types'
+
+import {
+  generateNewAttachmentResponse,
+  generateNewCheckboxResponse,
+  generateNewSingleAnswerResponse,
+  generateNewTableResponse,
+} from 'tests/unit/backend/helpers/generate-form-data'
+
+import { ProcessedSingleAnswerResponse } from '../../submission.types'
+import * as EmailSubmissionService from '../email-submission.service'
+
+const ALL_SINGLE_SUBMITTED_RESPONSES = [
```

Comment:
should filter through all fields to get the single fields so when we add new fields these tests will automatically cover them